common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java

```diff
@@ -94,16 +95,35 @@ public void onScreenInit(ScreenInitEvent e) {
         }
     }
 
+    private void getCustomJumpDestinations() {
+        String configDestinations;
+
+        switch (currentContainer) {
+            case BANK -> configDestinations = bankDestinations.get();
+            case BOOKSHELF -> configDestinations = bookshelfDestinations.get();
+            case MISC_BUCKET -> configDestinations = miscBucketDestinations.get();
+            default -> {
+                return;
+            }
+        }
+
+        customJumpDestinations = parseStringToDestinations(configDestinations);
+
+        if (customJumpDestinations == null) {
+            customJumpDestinations = QUICK_JUMP_DESTINATIONS;
```

Comment:
This is the correct default only for bank pages, not housing island containers, right?

Reply:
Yes, I didn't want to repeat another switch statement and in theory this should only fail if someone manually edited their config file as (hopefully) the onConfigUpdate should catch any non valid inputs

Reply:
Can change it to

```
switch (currentContainer) {
    case BANK -> customJumpDestinations = QUICK_JUMP_DESTINATIONS;
    case BOOKSHELF, MISC_BUCKET -> customJumpDestinations = HOUSING_DEFAULT_DESTINATIONS;
}
```

with `private static final List<Integer> HOUSING_DEFAULT_DESTINATIONS = List.of(1, 3, 4, 6, 8, 10);`?

Reply:
Should be good to go now unless you have some more improvements to suggest :)